modules/lazy-images/lazy-images.php

```diff
@@ -122,6 +122,11 @@ static function process_image_attributes( $attributes ) {
 			return $attributes;
 		}
 
+		// check for gazette featured images, which are incompatible
+		if ( isset( $attributes['class'] ) && strpos( $attributes['class'], 'gazette-featured-content-thumbnail' ) > 0 ) {
```

Comment:
You should use `false !== strpos()` instead to avoid the situation where `gazette-featured-content-thumbnail` is the first thing in the string.